Lib/unittest/async_case.py

```diff
@@ -115,11 +115,14 @@ def _callMaybeAsync(self, func, /, *args, **kwargs):
     def _setupAsyncioRunner(self):
         assert self._asyncioRunner is None, 'asyncio runner is already initialized'
         runner = asyncio.Runner(debug=True)
+        asyncio.set_event_loop(runner.get_loop())
         self._asyncioRunner = runner
 
     def _tearDownAsyncioRunner(self):
         runner = self._asyncioRunner
         runner.close()
+        asyncio.set_event_loop(None)
+        asyncio.set_event_loop_policy(None)
```

Comment:
Did this happen in 3.10?

Reply:
It was debugging leftover and PR is still draft as I am still working on this.